README.md

```diff
@@ -273,6 +273,66 @@ The pulsar-manager can monitor topics and subscriptions.
 
 ![pulsar-manager-token](docs/img/pulsar-manager-token.gif)
 
+## Casdoor
+
+
+### Casdoor Installation
+
+You can use casdoor to realize sso.
+
+Casdoor can connect to Pulsar-manager simplely.
+
+Because the code for connecting casdoor have been add in Pulsar-manager,so we just need configure application.yml in back-end and open front switch.
```

Comment:
```suggestion
Because the code for connecting the casdoor has been added in Pulsar-manager, we need to configure the casdoor in the back-end and front-end.
```